src/shared/bg_pmove.cpp

```diff
@@ -5229,18 +5228,18 @@ void PM_StepEvent( const vec3_t from, const vec3_t to, const vec3_t normal )
 	}
 }
 
-void slide( vec3_t wishdir, float wishspeed, playerState_t &ps )
+void pml_t::slide( vec3_t wishdir, float wishspeed, playerState_t &ps )
```

Comment:
This is not a good thing to make a member function. pml_t is a hodgepodge of variables which doesn't correspond to any coherent concept, so it is not a good fit for OOP techniques (consider e.g. the "single-responsibility principle").

Reply:
Fixed.